src/components/custom/dashboard/shared/chat/RealTimeChatInterface.jsx

```
@@ -0,0 +1,721 @@
+    const userRole = session?.user?.role || 'client';
+    const basePath = userRole === 'trainer' ? '/trainer/dashboard/messages' : '/client/dashboard/messages';
+    router.push(`${basePath}/${conversation.id}`);
+  };
+
+  return (
+    <div
+      className={`cursor-pointer border-b border-[#333] p-4 transition-colors hover:bg-[#1a1a1a] ${
+        isSelected ? "bg-[#1a1a1a]" : ""
+      }`}
+      onClick={handleClick}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)